sqlite3_opt_math_functions_test.go

```diff
@@ -0,0 +1,28 @@
+// +build sqlite_math_functions
+
+package sqlite3
+
+import (
+	"database/sql"
+	"testing"
+)
+
+func TestMathFunctions(t *testing.T) {
+	db, err := sql.Open("sqlite3", ":memory:")
+	if err != nil {
+		t.Fatal("Failed to open database:", err)
+	}
+	defer db.Close()
+
+	queries := []string{
+		`SELECT acos(1)`,
+		`SELECT log(10, 100)`,
+		`SELECT power(2, 2)`,
+	}
+
+	for _, query := range queries {
+		if _, err := db.Exec(query); err != nil {
```

Comment:
You should use `db.QueryRow` for `SELECT`, not `db.Exec`.

Reply:
I also considered `QueryRow`, but it defers the error to `Scan`, and the result isn't really needed for the test. Maybe `Query` would be a better idea?